### PR TITLE
Update local docker page for new war mounting approach

### PR DESCRIPTION
**Pull request by KieranFrance**
Update local docker deployment to detail how to mount the war file for use

### CODE REVIEW

**Review by Amy1612 — APPROVED** [review]

File: _includes/cloud-docker/localdocker.md [hunk 1]
```diff
@@ -7,20 +7,23 @@
...
-```
-
-Full version:
-
-{% capture docker-product %}{% if page.product == 'BuildVu' %}<[buildvu|buildvu-html|buildvu-svg]>{% elsif page.product == 'JPedal' %}<jpedal>{% endif %}{% endcapture %}
-```
-docker run -p 80:80 --env LICENSE_USERNAME=<YOUR USERNAME> --env LICENSE_PASSWORD=<YOUR PASSWORD> --env PRODUCT={{docker-product}} idrsolutions/{{ page.product | downcase }}
-```
+ 1. [Install Docker](https://docs.docker.com/get-started/#install-docker-desktop), then run the container like this:
+ 2. Download the {{ page.product }}-microservice.war file
```
Comment (daniel-warren) [inline _includes/cloud-docker/localdocker.md:11]:
```suggestion
 2. Download the {{ page.product | downcase }}-microservice.war file  
```
I suggest making sure that the war file described is in downcase

File: _includes/cloud-docker/localdocker.md [hunk 2]
```diff
@@ -7,20 +7,23 @@
...
-docker run -p 80:80 --env TOKEN=<TRIAL TOKEN>  idrsolutions/{{ page.product | downcase }}
-```
-
-Full version:
-
-{% capture docker-product %}{% if page.product == 'BuildVu' %}<[buildvu|buildvu-html|buildvu-svg]>{% elsif page.product == 'JPedal' %}<jpedal>{% endif %}{% endcapture %}
-```
-docker run -p 80:80 --env LICENSE_USERNAME=<YOUR USERNAME> --env LICENSE_PASSWORD=<YOUR PASSWORD> --env PRODUCT={{docker-product}} idrsolutions/{{ page.product | downcase }}
-```
+ 1. [Install Docker](https://docs.docker.com/get-started/#install-docker-desktop), then run the container like this:
```
Comment (daniel-warren) [inline _includes/cloud-docker/localdocker.md:10]:
```suggestion
 1. [Install Docker](https://docs.docker.com/get-started/#install-docker-desktop)
```
Running the container no longer makes sense in context

File: _includes/cloud-docker/localdocker.md [hunk 3]
```diff
@@ -7,20 +7,23 @@
...
+    - **For trial users:**
+      - If you haven't already, sign up for a [{{ page.product }} trial](https://www.idrsolutions.com/{{ page.product | downcase }}/trial-download).  
+      - Open the email you received when you signed up.  
+      - Follow the download link found in the docker section.
+      
+    - **For customers:**
+      - Visit the [{{ page.product }} customer download page](https://www.idrsolutions.com/{{ page.product | downcase }}/download).  
+      - Click on 'Download Latest Customer WAR'.  
+      - Enter your username and password.  
+      - Click submit.
```
Comment (daniel-warren) [inline _includes/cloud-docker/localdocker.md:21]:
```suggestion
      - Submit your username and password.  
```
Suggest updating similar to the [suggestions](https://github.com/idrsolutions/idrsolutions.github.io/pull/3#pullrequestreview-504046616) in other PRs

**Review by daniel-warren — CHANGES_REQUESTED** [review]
Looks promising, have suggested a few changes.

File: _includes/cloud-docker/localdocker.md [hunk 4]
```diff
@@ -7,20 +7,23 @@
...
+      
+    - **For customers:**
+      - Visit the [{{ page.product }} customer download page](https://www.idrsolutions.com/{{ page.product | downcase }}/download).  
+      - Click on 'Download Latest Customer WAR'.  
+      - Enter your username and password.  
+      - Click submit.  
+      
+ 3. Run the following command  
+ ```
+ docker run -p 80:80 ---mount "source=/path/to/war/{{ page.product | downcase }}_microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions/{{ page.product | downcase }}
```
Comment (leonatherton) [inline _includes/cloud-docker/localdocker.md:25]:
```suggestion
 docker run -p 80:80 ---mount "source=/path/to/war/{{ page.product | downcase }}-microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions/{{ page.product | downcase }}
```

**Review by leonatherton — CHANGES_REQUESTED** [review]

File: _includes/cloud-docker/localdocker.md (hunk 1 above)
Comment (KieranFrance) [reply _includes/cloud-docker/localdocker.md:11]: Updated in next commit

File: _includes/cloud-docker/localdocker.md (hunk 2 above)
Comment (KieranFrance) [reply _includes/cloud-docker/localdocker.md:10]: Updated in next commit

File: _includes/cloud-docker/localdocker.md (hunk 3 above)
Comment (KieranFrance) [reply _includes/cloud-docker/localdocker.md:21]: Updated in next commit

File: _includes/cloud-docker/localdocker.md (hunk 4 above)
Comment (KieranFrance) [reply _includes/cloud-docker/localdocker.md:25]: Updated in next commit

**Review by daniel-warren — APPROVED** [review]

**Review by leonatherton — APPROVED** [review]

**Pull request merged**